kernel/contracts/KernelRegistry.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.4.18;
+
+import "./KernelInstance.sol";
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract KernelRegistry is Ownable {
+  event NewInstance(KernelInstance indexed instance);
```

Comment:
Let's add name (indexed and non-indexed) and version (non-indexed) to the event, so a client can monitor for new releases of a distribution

Reply:
Same limitation as above in recovering `name` and `version` from `KernelInstance`.. There are some strategies around to map strings to fixed-sized arrays, we could maybe go with such an approach if needed..

Reply:
Good point. I'm thinking about moving `name` and `version` out of the KernelInstance contract, and have the registry store instances as a `mapping(bytes32 => KernelEntry)`, where `KernelEntry` is a struct with `name`, `version` and `address`.

This way, name and version are managed entirely by the registry contract. WDYT? It solves the issue, though I'm not sure I like having to go with a worse design just to overcome a language limitation.

Reply:
Hmm.. On the plus side, doing this we would be able to issue crucial information in the events, plus we don't delegate the hash computation. I don't know if there's any downside other than a Kernel not knowing its name or version.. 

Maybe it's a bit strange that it would still know its `developer` (set in constructor) and `parent` (also set there, plus used copiously in the logic..) (unless we change these too, but all becomes more cumbersome). 

I tend to resist the worse design but also cannot see any practical issues for not doing this, @facuspagnuolo care to chip in?